BankApp/BankLibrary/IBank.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace BankLibrary
+{
+    public interface IBank<out T>
+    {
+        public void OpenAccount(OpenAccountParameters parameters);
```

Comment:
drop access modifiers
```suggestion
        void OpenAccount(OpenAccountParameters parameters);
```